src/systems/elevator/vender/metapushkin/LICENSE

```diff
@@ -0,0 +1,201 @@
+		       The Artistic License 2.0
```

Comment:
@j-rivero , do you see any issues mixing this license here?

Reply:
The Artistic License 2.0 is compatible with Apache 2.0